frontend/src/app/[locale]/stocks/[ticker]/components/StockDetailsLayout.tsx

```diff
@@ -19,7 +14,9 @@ import { useGetStockDetails } from "@/query/stock-details";
 import { useStockTransaction } from "@/query/transaction";
 import { createPriceHandler, stockUpdateClient } from "@/ws";
 import type { AxiosError } from "axios";
+import { ExternalLinkIcon } from "lucide-react";
 import { useLocale, useTranslations } from "next-intl";
+import Link from "next/link";
```

Comment:
```suggestion
import Link from "@/i18n/routing";
```
it doesnt really matter in this case but let's be consistent everywhere. since we're not using to to navigate within the app it would be probably better to use `<a>` tag.